src/components/theme-switcher/user-dropdown-switcher/index.tsx

```diff
@@ -0,0 +1,20 @@
+import ThemeSwitcher from 'components/theme-switcher'
+import {
+	DropdownDisclosureListItem,
+	DropdownDisclosureSeparatorItem,
+} from 'components/dropdown-disclosure'
+import s from './user-dropdown-switcher.module.css'
+
+export default function UserDropdownDisclosureThemeSwitcher() {
+	return (
+		<>
+			<DropdownDisclosureSeparatorItem />
+			<DropdownDisclosureListItem className={s.listItem}>
+				<label className={s.label} htmlFor="theme-switcher">
+					Theme
+				</label>
+				<ThemeSwitcher id="theme-switcher" />
```

Comment:
📌 💭 A breadcrumb for subsequent PRs: we'll want to make sure that all each of switchers have a unique ID, since there'll be multiple rendered on a single page.

Reply:
i'll make this one for the dropdown nav more specific! good point.